index.html

```diff
@@ -35,12 +39,199 @@ <h1 class="title">
                 </p>
             </article>
 
-            <button class="text btn">
+            <button class="buttons btn">
                 Start Collaboration
             </button>
 
             <img id="geometry" src="./images/Thur.svg" alt="">
         </section>
+
+        <section class="works">
+            <h2 class="h2">Projects</h2>
+            <article class="portfolio">
+                <img src="./images/one.svg" alt="project one is a web site" class="project__photo">
+                <div class="project__info">
+                    <h3 class="project__title">Hello Microverse</h3>
+                    <div class="project__stack__container">
+                        <p class="project__stack">HTML/CSS</p>
+                        <p class="project__stack">Ruby on rails</p>
+                        <p class="project__stack">JavaScript</p>
+                    </div>
+                    <div class="project__button__holder">
+                        <button class="buttons project__button">
+                            See this project
+                            <img src="./images/Union.svg" alt="" class="arrow">
+                        </button>
+                    </div>
+                </div>
+            </article>
+            <article class="portfolio">
+                <img src="./images/two.svg" alt="project one is a web site" class="project__photo">
+                <div class="project__info">
+                    <h3 class="project__title">Hello Microverse</h3>
+                    <div class="project__stack__container">
+                        <p class="project__stack">HTML/CSS</p>
+                        <p class="project__stack">Ruby on rails</p>
+                        <p class="project__stack">JavaScript</p>
+                    </div>
```

Comment:
- [ ] The list of project technologies (e.g., "JavasScript", "HTML" and "CSS") are a list `ul` and each of them a `li`, please kindly fix that 🙃 🙃